### ISSUE TITLE
[BUG] Create two `hqps_actor_autogen` targets cause cmake error

### ISSUE THREAD

**Issue by zhanglei1949** [alibaba/hiactor#30]
As I create two `hqps_actor_autogen` targets in my CMakeLists.txt, cmake error occurs
```bash
CMake Error at /usr/local/bin/hiactor_codegen/ActorAutoGen.cmake:42 (add_custom_target):
  add_custom_target cannot create target "installed_libclang" because another
  target with the same name already exists.  The existing target is a custom
  target created in source directory
  "/root/GraphScope/balabala/graph_db".  See documentation for policy
  CMP0002 for more details.
Call Stack (most recent call first):
  engines/balabala/CMakeLists.txt:39 (include)
```

**Comment by LiSu** [comment]
Closed by #31

**Comment by LiSu** [comment]
Closed by #31
